### ISSUE TITLE
As a user, I should not be able to click through to table views from search results

### ISSUE THREAD

**Issue by robredpath** [OpenDataServices/grantnav#203]

**Comment by robredpath** [comment]
It may be better to rename the links to a 'see also'

**Comment by mariongalley** [comment]
I think that makes sense - would you then only be able to navigate to funder & recipient pages from the funders & recipients tables linked on the home page? And would you be able to get to location pages at all in this case @robredpath ?

**Comment by robredpath** [comment]
@mariongalley I'm not sure that we want to do this now, given that we're doing some work to make those pages more useful! I do wonder about making it a bit clearer what the links on the right-hand side of results cards mean, though - maybe through a logo or some microcopy? I'm not sure exactly what, but...something.

**Comment by mariongalley** [comment]
Yeah I think those links should say something like "<Funder name> more..." or something so that there's a more/see more/more about/details Call to Action indicating that by clicking on the link you're opening a page about that funder/location/recipient rather than the grant card you're on.

**Comment by mariongalley** [comment]
I think we've sufficiently improved the labelling on the cards to solve this issue
